src/chaat/config.clj

```diff
@@ -0,0 +1,28 @@
+(ns chaat.config
+  (:require [clojure.java.io :as io]
+            [clojure.edn :as edn])
+  (:import (java.io PushbackReader)))
```

Comment:
[question] Why are we using a `PushbackReader` specifically?

Reply:
+1

Reply:
[read](https://clojuredocs.org/clojure.edn/read) which I'm using to parse my `.edn` config file seems to specifically require a `PushbackReader`